.github/workflows/build.yml

```
@@ -38,3 +38,25 @@ jobs:
       - name: Build docker image
         run: |
           docker build .
+
+  build-proto:
+    runs-on: ubuntu-20.04
```

You can use the version `ubuntu-22.04` in order to be aligned with the other runners versions.

```suggestion
    runs-on: ubuntu-22.04
```